blockchain/ethereum.go

```diff
@@ -839,7 +839,12 @@ func (e *EthereumClient) EstimateGas(callMsg ethereum.CallMsg) (GasEstimations,
 		gasTipCap, err = e.Client.SuggestGasTipCap(ctx)
 		cancel()
 		if err != nil {
-			return GasEstimations{}, err
+			//For Besu, that support "eth_maxPriorityFeePerGas" https://github.com/hyperledger/besu/issues/5658
```

Comment:
Does Besu just make the estimation for us in this case @Tofel ?

Reply:
Nope, we just set it to `0` and use just the base fee. But it won't work. I just tried Besu with contract upload and there we use `go-ethereum` client code to do the upload, which fails, because that method is not implemented. So we will need to work with Besu in non-EIP1559 mode until they have implemented `eth_maxPriorityFeePerGas` (or implement it ourselves in the upstream? ;-))